app/src/main/java/grinnell/appdev/edu/lyles/Adapters/BeerAdapter.java

```diff
@@ -36,14 +42,20 @@ public View getView(int pos, View convertView, ViewGroup parent) {
             convertView = LayoutInflater.from(this.getContext()).inflate(R.layout.beer_list_item, parent, false);
         }
 
-        final BeerItem beer = BeerAdapter.super.getItem(pos);
+        BeerItem beer = BeerAdapter.super.getItem(pos);
 
         if (beer != null) {
-            // create references to view items
-            // final ImageView beerImage = convertView.findViewById(R.id.beerImage);
-        }
+            Log.d("beer", "Beer item is not null");
```

Comment:
Please don't leave log statements in code pushed out into production. Log statements should only be on your local machine.